src/System.Windows.Forms/src/System/Windows/Forms/Controls/Buttons/CheckBox.CheckBoxAccessibleObject.cs

```diff
@@ -91,5 +92,47 @@ internal override void Toggle()
                 }
             }
         }
+
+        internal override void SetValue(string? newValue)
+        {
+            if (this.TryGetOwnerAs(out CheckBox? owner))
```

Comment:
Please invert the condition to reduce nesting.
```suggestion
            if (!this.TryGetOwnerAs(out CheckBox? owner))
            {
                return;
            }
```